src/main/java/net/dv8tion/jda/internal/utils/Checks.java

```diff
@@ -38,7 +38,7 @@
 public class Checks
 {
     public static final Pattern ALPHANUMERIC_WITH_DASH = Pattern.compile("[\\w-]+", Pattern.UNICODE_CHARACTER_CLASS);
-    public static final Pattern ALPHANUMERIC = Pattern.compile("[\\w]+", Pattern.UNICODE_CHARACTER_CLASS);
+    public static final Pattern ALPHANUMERIC = Pattern.compile("\\w+", Pattern.UNICODE_CHARACTER_CLASS);
```

Comment:
Why the change?

Reply:
It was redundant and gave annoying inspection warnings